docs/changes/cosmology/12862.api.rst

```diff
@@ -0,0 +1,2 @@
+The ``fvalidate="float"`` option for a Parameter is deprecated in favor of
+``dtype=float``, and will be removed in at least two minor versions (>v5.3).
```

Comment:
Maybe just deprecate in 6.0? Or even next LTS? Means people who support both LTS and most recent do not have to do version checks.

Reply:
SGTM!